docs/contributors/check-accessibility.md

```diff
@@ -0,0 +1,179 @@
+- Meaning is conveyed through text or icons, not by color alone.
+    * For example: for errors, an error message is shown in addition to a red border.
+- Animations are disabled for users that have {guilabel}`Animation Effects` disabled in the Settings app ({menuselection}`Settings --> Accessibility --> Seeing`).
+- No flashing (there should be less than 3 flashes per second)
+- All elements have enough contrast with their background
+    * Read [Contrast checks](#contrast-checks) for further guidance.
+
+### Contrast checks
+
+Although contrast is usually handled at the theme level, you should ensure all elements have enough color contrast so all text and controls are perceivable:
+
+- For text: 4.5:1 or higher contrast with background
+- For everything else (for instance, icons on background): 3:1 or higher contrast with background
+
+For checking **color contrast**, install a checker such as [Kontrast](https://snapcraft.io/kontrast) or [Contrast](https://flathub.org/apps/org.gnome.design.Contrast), and pick the foreground and background colors.
+
+You should be particularly cautious in the following cases:
+
+- Check how your local style overrides affect states like hover or focus
+- Check light colors, such as light gray for secondary text or yellow
+- Check semantic colors for text, such as red for errors or yellow for warnings. Compare light and dark theme
+
+No minimum contrast is required for inactive components, decorative images or graphics, or logos.
+
+## Check keyboard navigation
+
+These checks ensure a good experience for keyboard-only users, including people with motor disabilities that cannot use a mouse or trackpad, blind users who rely on screen readers, and power users who just prefer to use their keyboard.
+
+### Try keyboard shortcuts
+
+For a simple test, select the window you want to test. Then hit the {kbd}`Tab` key repeatedly and check how focus moves through all interactive elements.
+
+For a more thorough test, you need these shortcuts:
+
+| Action | Shortcut |
+|--------|----------|
+| Move to next interactive item | {kbd}`Tab` |
+| Move to previous interactive item | {kbd}`Shift+Tab` |
+| Interact with an item or group | {kbd}`Enter` |
+| Exit a group | {kbd}`Esc` |
+| Move between windows/main area, top bar and dashboard | {kbd}`Ctrl+Tab` |
+
+### Keyboard navigation checklist
+
+- All interactive elements (buttons, fields, links…) are focusable by using the {kbd}`Tab` key
+- All interactive elements have a visible focus state (ideally a focus ring)
+- There are no focus traps: it is possible to escape all loops, for instance by hitting the {kbd}`Esc` key
+- Focus moves from one element to the next in a logical order
+    * In most of our apps, that usually means sidebar (if open), then body; and top to bottom, left to right (or right to left for RTL languages such as Arabic or Hebrew).
+- Focus moves to modals/dialogs when opened, and returns to the trigger when closed
```

Comment:
What's a modal in this context?

Reply:
(might be worth including a glossary entry if you don't want to define it here)

Reply:
Ooops, missed this one. I dropped "modal" from there, dialog is more understandable and used also in [GNOME's Human Interface Guidelines](https://developer.gnome.org/hig/patterns/feedback/dialogs.html).